dashboard/api_test.py

```diff
@@ -1254,15 +1254,19 @@ def test_format(self, mock_cache_refresh):
         for cache_type in CachedEdxDataApi.SUPPORTED_CACHES:
             mock_cache_refresh.assert_any_call(self.user, self.edx_client, cache_type)
 
-        assert len(result) == 2
+        assert isinstance(result, dict)
```

Comment:
I usually try to check the structure of the dictionary in the tests so if it fails because it is different I know why and I have not infer from the error.
I have no strong opinions about this either.